src/workloads/execution/VariadicAggregateExpressions.yml

```diff
@@ -54,6 +39,9 @@ GlobalDefaults:
                                                  {case: {$gt: ["$int8", 100]}, then: "$int8"},
                                                  {case: {$gt: ["$int9", 100]}, then: "$int9"},
                                                  {case: {$gt: ["$int10", 100]}, then: "$int10"}]
+  IntGenerator: &IntGenerator {^Join: {array: [{ ^FormatString: {format: "%03d", withArgs: [{^Inc: {start: 1}}]}}]}}
```

Comment:
one problem that I ran into with ^Inc is that for using it repeatedly, you will keep getting higher and higher values ([see comment here](https://github.com/mongodb/genny/blob/b41ef95d4870c8d375a0ae3f7a99b44bd02bb643/src/workloads/execution/FilterWithComplexLogicalExpression.yml#L98-L111)). Before merging, please verify that this produces the fields that you expect.

Reply:
I was running into that issue initially. I used ^FixedGeneratedValue to generate the list of ints, so it is generated only once.